data/BDFL669.json

```diff
@@ -1,21 +1,69 @@
 {
   "name": "Nigel Lowa",
   "bio": "A software protoge with a penchant for developing impressional applications",
+  "tags": [
+    "Open Source",
+    "JavaScript",
+    "React"
+  ],
+  "socials": [
+    {
+      "icon": "FaTwitter",
+      "url": "https://twitter.com/botyliciouss"
+    },
+    {
+      "icon": "FaGithub",
+      "url": "https://github.com/BDFL669"
+    },
+    {
+      "icon": "FaLinkedin",
+      "url": "https://www.linkedin.com/in/nigel-okoth-104609206/"
+    }
+  ],
+  "testimonials": [
+
+  ],
   "links": [
     {
-      "name": "Follow me on Twitter",
+      "name": "Follow me on GitHub",
+      "group": "Socials Primary",
+      "name": "Twitter: Follow me",
       "url": "https://twitter.com/botyliciouss",
-      "icon": "FaTwitter"
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
     },
     {
-      "name": "Follow me on GitHub",
-      "url": "https://github.com/lowaCase",
-      "icon": "FaGithub"
+      "group": "Socials Primary",
+      "name": "GitHub: Collaborate on Open Source",
+      "url": "https://github.com/BDFL669",
+      "icon": "FaGitHub"
     },
     {
-      "name": "Connect with me on LinkedIn",
-      "url": "https://www.linkedin.com/in/nigel-okoth/",
+      "group": "Socials Primary",
+      "name": "LinkedIn: Let's connect",
+      "url": "https://www.linkedin.com/in/nigel-okoth-104609206/",
       "icon": "FaLinkedin"
     }
+  ],
+  "milestones": [
+    {
+      "title": "Started Blogging on Dev.to",
+      "date": "Feb 2022",
+      "icon": "SiHashnode",
+      "color": "Blue",
+      "description": "Started Blogging on Dev.to",
+      "url": "https://dev.to/anvilicious"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/botyliciouss",
+      "icon": "FaTwitter"
```

Comment:
Please add a `,` here.

Reply:
Thank you for your feedback. I have made the necessary changes and validated them on the playground